akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -462,7 +463,7 @@ object ClusterEvent {
       }
 
       val removedMembers = oldGossip.members diff newGossip.members
-      val removedEvents = removedMembers.map(m ⇒ MemberRemoved(m.copy(status = Removed), m.status))
+      val removedEvents = removedMembers.unsorted.map(m ⇒ MemberRemoved(m.copy(status = Removed), m.status))
```

Comment:
we might want to look into how the performance of `.unsorted` compares to `.toSet` and `.iterator.map...toSet`

Reply:
for immutable `SortedSet`s it's `override def unsorted: Set[A] = this`, so that sounds good? Apparently the `SortedSetOps` `map` does not override the one from `Set`, but is selected at compile time.